userguide/one_shot_object_detection/README.md

```diff
@@ -0,0 +1,120 @@
+# One-Shot Object Detection
+
+One-Shot object detection is the task of learning to detect from as few as one example per category. Unlike the Object Detector which requires many varied examples of objects in the real world, the One-Shot detector only requires a single canonical example of an object to train the classifier on and performs predictions for that category in the wild. It is best suited for 2D objects or when there isn't much expected variance in the examples, such as playing cards, logos, road signs and clapperboards. One-Shot detector is not suitable for 3D objects such as faces, animals and cars. 
+
+Given an image and a one-shot detector trained on category **stop sign**, the output prediction will look like:
+
+![Stop Sign vector and prediction images](images/stop_sign_sample.jpg) 
+
+
+#### Introductory Example
+
+In this example, the goal is to **localize** where in the image instances of the category **stop sign** are present. For this task we supply a single image of a stop sign as a starter image:
```

Comment:
How about, "In this example, our goal is to **localize** instances of **stop signs** in images. For this application, we supply a single image of a stop sign as a starter image:".

Reply:
I like the edit.